_Boards/Atmel/Board_ProMicro/MFBoards.h

```diff
@@ -1,21 +1,20 @@
 #ifndef MFBoardMicro_h
 #define MFBoardMicro_h
 
-#define MF_SEGMENT_SUPPORT 0
-#define MF_LCD_SUPPORT 0
-#define MF_STEPPER_SUPPORT 0
-#define MF_SERVO_SUPPORT 0
+#define MF_SEGMENT_SUPPORT 1
+#define MF_LCD_SUPPORT 1
+#define MF_STEPPER_SUPPORT 1
+#define MF_SERVO_SUPPORT 1
 #define MF_ANALOG_SUPPORT 1
-#define MF_SHIFTER_SUPPORT 0
-#define MF_INPUT_SHIFTER_SUPPORT 1
+#define MF_SHIFTER_SUPPORT 1
 
 // ALL          26892 (93.8%) / 1713 (66.9%)
 // No Segments  25148 (87.7%) / 1580 (61.7%)
 // No LCDs      25308 (88.3%) / 1678 (65.5%)
 // NO Steppers  21892 (76.4%) / 1675 (65.4%)
 // No Servos    25302 (88.2%) / 1620 (63.3%)
 
-#define MODULE_MAX_PINS 25
+#define MODULE_MAX_PINS 21
```

Comment:
Another error on my part making edits in the wrong branch. This should be 21 as listed here: https://github.dev/MobiFlight/MobiFlight-Connector/blob/51613650fa8142f4a7338633dac631e2ec33fb2a/FirmwareSource/mobiflight/mobiflight.ino#L88